src/core/Akka/Actor/Cell.cs

```diff
@@ -73,9 +73,11 @@ public interface ICell
         void Post(IActorRef sender, object message);
 
 
-
+        [Obsolete("Used ChildrenRefs instead")]
```

Comment:
`ChildrenRefs` was `private` and was, itself, marked as `Obsolete`. I decided just to use the `ChildContainer` instead.